pallets/funding/src/lib.rs

```diff
@@ -1232,7 +1253,7 @@ pub mod pallet {
 			Self::do_end_funding(project_id)
 		}
 
-		#[pallet::call_index(33)]
+		#[pallet::call_index(36)]
```

Comment:
Shouldn't this be 35, and then 36?

Reply:
We added `remove_project` as 35. So I see three solutions:
1. Keep it like this, so 36 instead of 35
2. Change index of remove_project to some other value
3. actually reindex everything but not sure if that will break things.

Reply:
I would reindex everything. Better do it know when there's not any big repercusions @lrazovic ?